plio/settings.py

```diff
@@ -264,7 +264,7 @@
 API_APPLICATION_NAME = "plio"
 
 OAUTH2_PROVIDER = {
-    "ACCESS_TOKEN_EXPIRE_SECONDS": 60 * 60 * 24,  # 1 day
+    "ACCESS_TOKEN_EXPIRE_SECONDS": 60 * 60 * 24 * 7,  # 7 days
```

Comment:
after checking the repo, seems like only users logging in through OTP will be affected by this. 
If we cannot change the expiry time of google based logins too, I think keeping the same expiry time across all auth types is better, no?

Reply:
No.
It works for google based logins too. 
The `ACCESS_TOKEN_EXPIRE_SECONDS` that I changed here is what is being used by both our custom OTP code and also by the rest_framework_social_oauth2 package.
I tested it out and it sets an expiry time of 7 days.